meta-resin-common/recipes-core/util-linux/util-linux_%.bbappend

```diff
@@ -21,3 +21,9 @@ python() {
 		if packages:
 			d.setVar('PACKAGES', 'util-linux-lsblk' + ' ' + packages)
 }
+
+ALTERNATIVE_${PN} += "fallocate fsfreese nsenter unshare"
+ALTERNATIVE_LINK_NAME[fallocate] = "${bindir}/fallocate"
+ALTERNATIVE_LINK_NAME[fsfreese] = "${sbindir}/fsfreeze"
+ALTERNATIVE_LINK_NAME[nsenter] = "${bindir}/nsenter"
+ALTERNATIVE_LINK_NAME[unshare] = "${bindir}/unshare"
```

Comment:
Is it possible to remove these via busybox's defconfig?

Reply:
That would be another option indeed but this fix would actually make sense in upstream too so I'll keep it with the intention to upstream it.